app/views/facilities/index.html.haml

```diff
@@ -6,6 +6,21 @@
 
 .alert.alert-info= text(".welcome")
 
+- if @products.any?
+  %h4= text(".recently_purchased")
+  .product_list{ class: @products.first.class.model_name.to_s.pluralize.downcase }
```

Comment:
Is there a reason we can’t use the `product_list` partial itself?

Part of it may be related to sorting, and if that’s the case, we could pull the sorting out of that and do it in `facilities/show`, though to be honest, I think we should sort the products alphabetically. I think it’ll be easier to use that way. 

If we do decide to decide to leave this here, there’s probably some cleanup we can do: I don’t think the classes here actually do anything.

It might also be nice to add the facility name. But I could go either way.

Also, as noted in the ticket, we want to show recent facilities first, then products.

Reply:
The `product_list` partial expects a current_facility, which we don't have in this context. Is there an easy workaround that would let us use the partial anyway?

Sorting alphabetically seems fine to me.  What do you mean about adding the facility name?

Reply:
Add the facility name to the product link?  Something like: `Product name (Facility name)`?

Reply:
So facilities have a `sorted` scope and products have an `alphabetized` scope, they do very similar things.  I lean towards `alphabetized` and `sorted` is only used in a handful of places.  Do we want to change this or leave it alone?